client/services/FeedbackGeneratorService.js

```diff
@@ -47,11 +45,11 @@ tie.factory('FeedbackGeneratorService', [
     var UNFAMILIARITY_THRESHOLD = 5;
 
     /**
-     * Counter to keep track of consecutive syntax errors.
+     * Counter to keep track of language unfamiliarity errors.
```

Comment:
We should probably add a note to this JSDoc comment that a "language unfamiliarity" error includes syntax errors and wrong language errors.